wiki/CreatingInterfaces.md

```diff
@@ -242,11 +245,14 @@ If `color` is defined but either the inactive or hover color is not, the undefin
 		from <x#> <y#> to <x#> <y#> [<anchor>]
 		color <color>
 		size <size#>
+		[reversed]
+		[start angle <angle#>]
+		[span angle <angle#>>]
 ```
 Defines a straight line (bar) or circular outline (ring) to be drawn with this interface.
-In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box.
+In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box. The start and span angle specifies the portion of the ring that is drawn.
```

Comment:
```suggestion
In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box. The start and span angle specifies the portion of the ring that is drawn **(v. 0.10.2)**.
```
https://github.com/endless-sky/endless-sky/commit/b0beb6e664d0de38d9a949521825201727114d09